config/default.js

```diff
@@ -30,17 +30,22 @@ module.exports = {
   darServer: {
```

Comment:
I saw you removed the dar stuff .... should this go too?

Reply:
Yes!

Reply:
Actually, the DAR server package which uses this is still in there. I'll remove it in a separate PR.

Reply:
Agreed